lib/ex_stone_openbank/config.ex

```diff
@@ -40,8 +40,8 @@ defmodule ExStoneOpenbank.Config do
 
   # we change the key name here
   defp validate(:private_key, private_key) do
-    unless String.contains?(private_key, "-----BEGIN PRIVATE KEY-----") and
-             String.contains?(private_key, "-----END PRIVATE KEY-----") do
+    unless String.contains?(private_key, "-----BEGIN RSA PRIVATE KEY-----") and
```

Comment:
I don't know if this is really needed. First I removed this validation and removed "RSA" text from the key. An error was raised by `:public_key.pem_decode` and I stepped back using the "RSA" text

Reply:
This is actually a very bad way of validating the private key... the best thing to do here would be to try and just parse it. The PEM specification allows for metadata to be added previous to the key declaration for instance...

One way to enhance things here would be to drop the parsing to Joken/JOSE directly and tell if it is valid.

Reply:
Totally agree. Even the try/catch block seemed to be unnecessary, since it was just logging and raising the same error. In my terminal the result was an error thrown and a log with the same content